catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/FileSystemPersistenceProvider.java

```diff
@@ -107,14 +108,15 @@ public void store(String key, Object value) {
                 }
             }
             file = new FileOutputStream(getMapStorePath() + key + PERSISTED_FILE_SUFFIX);
-            OutputStream buffer = new BufferedOutputStream(file);
+            buffer = new BufferedOutputStream(file);
             output = new ObjectOutputStream(buffer);
             output.writeObject(value);
         } catch (IOException e) {
             LOGGER.debug("IOException storing value in cache with key = " + key, e);
         } finally {
```

Comment:
Same comment about removing the finally.

Reply:
Updated to use try with resource in each of these files.